Dockerfile

```diff
@@ -41,14 +46,15 @@ RUN pip install --no-cache-dir --upgrade pshtt==0.6.6
 ###
 # Install domain-scan
 ###
-RUN git clone https://github.com/18F/domain-scan /home/scanner/domain-scan/
+RUN git clone https://github.com/18F/domain-scan \
+    ${HOME}/domain-scan/
 RUN pip install --no-cache-dir --upgrade \
-    --requirement /home/scanner/domain-scan/requirements.txt
+    --requirement ${HOME}/domain-scan/requirements.txt
 
 ###
 # Remove build dependencies
 ###
-RUN apt-get remove --quiet --quiet ${BUILD_DEPS}
+RUN apt-get remove --quiet --quiet $BUILD_DEPS
```

Comment:
Was this supposed to be INSTALL_DEPS?

Reply:
Great catch!  Fixed in 86f1ea3.
:baseball:

Reply:
Quality!

Reply:
> Quality!

[What is quality?](https://en.wikipedia.org/wiki/Pirsig%27s_Metaphysics_of_Quality)